package.json

```diff
@@ -26,8 +26,8 @@
     "season": "^6.0.2"
   },
   "configSchema": {
-    "isEnabled": {
-      "description": "Enable loading project configurations",
+    "isActive": {
+      "description": "Actively detect and load project configurations",
```

Comment:
I think you can take out the word "Actively" and just say "Detect and load project configurations"

Reply:
I feel "Detect and load project configurations" alone is like the nature of this package, not indicative of the ability to activate/deactivate it w/o disabling. How about "Detect and load project configurations automatically" or something like this?

Reply:
That sounds fine to me!